# timestamp, frame id
Header header

#set of targets being tracked
peopleTracking[] peopleSet